templates/terraform/environments/prod/user_access.tf

```diff
@@ -0,0 +1,111 @@
+# define AWS policy documents for developer
+data "aws_iam_policy_document" "developer_access" {
```

Comment:
At some point we will want to revisit these policies, add stuff like enforcing MFA, etc. but this should be fine for now.